.pylintrc

```diff
@@ -158,6 +152,8 @@ disable=abstract-method,
         # broken by putting imports inside functions so they aren't
         # executed upon module import.
         cyclic-import,
+        broad-exception-raised,
```

Comment:
Why are we adding new exceptions?

Reply:
This rule disallows raising exceptions of type `Exception`, which we do frequently in the code base. This must be a new lint rule introduced since we last upgraded pylint